jooq-common/src/main/java/net/furizon/jooq/generated/tables/Events.java

```diff
@@ -71,7 +71,7 @@ public Class<Record> getRecordType() {
     /**
      * The column <code>public.events.event_date_end</code>.
      */
-    public final TableField<Record, String> EVENT_DATE_END = createField(DSL.name("event_date_end"), SQLDataType.VARCHAR(32), this, "");
+    public final TableField<Record, String> EVENT_DATE_TO = createField(DSL.name("event_date_end"), SQLDataType.VARCHAR(32), this, "");
```

Comment:
Did you renamed it? Better to call plugin for the re-generation :D 
because you did forget to rename the DSL.name("event_date_end") part, so you will get an exception

Reply:
It was a dirty change because I didn't had access to a db to run the migration hahaha
Gonna do it asap!